content/developers.html

```diff
@@ -146,6 +146,27 @@ <h1 id="indicators">Developer Resources for Apache CloudStack</h1>
 
 <div class="col-lg-8">
 
+<h2>CloudStack Repositories</h2>
+<p><a href="https://github.com/apache/cloudstack" target="_blank">System Source Code</a>. The main CloudStack-related project, 
```

Comment:
these paths are worth mentioning but not the "origin of truth". gitbox.apache.org contains the repository and is sync'd to github!

Reply:
@DaanHoogland removed the part.